OFXParser/Core/Parser.cs

```diff
@@ -198,6 +199,23 @@ public static Extract GetExtract(String ofxSourceFile, ParserSettings settings)
                 meuXml.Close();
             }
 
+            foreach (Transaction tr in extrato.Transactions)
+            {
+                if (!(new List<string> { "CREDIT", "DEBIT" }).Contains(tr.Type))
```

Comment:
Suggestion: create private method to check this; and use it into Linq expression